platzky/seo/seo.py

```
@@ -14,30 +19,36 @@ def create_seo_blueprint(db, config: dict[str, t.Any], locale_func: t.Callable[[
         response.headers["Content-Type"] = "text/plain"
         return response
 
-    def get_blog_entries(host_base, lang, db, blog_prefix):
+    def get_blog_entries(
+        host_base: str, lang: str, db: t.Any, blog_prefix: str
+    ) -> list[dict[str, str]]:
+        """Generate sitemap entries for all blog posts.
+
+        TODO: Add get_list_of_posts for faster getting just list of it.
+        """
         dynamic_urls = list()
```

Ruff (0.14.10)

Unnecessary `list()` call (rewrite as a literal)

Rewrite as a literal

(C408)